README.md

```diff
@@ -135,20 +496,322 @@ the test case
 indicating that the test driver finished its execution normally, i.e., did not
 crash.
 
-# How to use DDT
 
-In its first implementation, Data Driven Test uses data files formatted with
-JSON structures describing tests and parameters. The data directory string is
-set up as follows:
+# How to update Conformance Testing: ICU versions, platforms, components
+
+Data Driven Testing is expected to remain current with ICU programs and data
+updates. It is also designed to support new testing platforms in addition to the
+current set of Dart, ICU4C, ICU4J, ICU4X, and NodeJS. And new types of tests,
+i.e., "components", may be added to Conformance testing.
+
+This section describes the process for keeping DDT up to date with needed test
+types and required programming platforms
+
+## Incorporating new ICU  / CLDR versions into DDT
+
+ICU releases are usually made twice each calendar year, incorporating new data,
+fixes, and new test files. ICU versions may also add new types of data
+processing. A recent example is Message Format 2.
+
+Because Data Driven Testing operations with multiple ICU and CLDR versions, this
+system should be updated with each new ICU release. Here are several pull
+requests for recent ICU updates:
+
+* [ICU 76 for C++](https://github.com/unicode-org/conformance/pull/325/)
 
-## A directory `testData` containing
-  * Test data files for each type of test, e.g., collation, numberformat,
-  displaynames, etc. Each file contains tests with a label, input, and
-  parameters.
-  * Verify files for each test type. Each contains a list of test labels and
-  expected results from the corresponding tests.
+* [ICU 76 for Java](https://github.com/unicode-org/conformance/pull/344)
 
-## Directory `testOutput`
+* [ICU76 for NodeJS](https://github.com/unicode-org/conformance/pull/348)
+
+### ICU4C updates
+
+These are usually the first changes to be made because ICU4C includes both code
+and test data updates for many components.
```

Comment:
```suggestion
These are usually the first changes to be made because ICU4C includes updates to code,
locale data, and test data for many components.
```